samples/sample_multi_transcode/src/transcode_utils.cpp

```diff
@@ -230,7 +230,7 @@ void TranscodingSample::PrintHelp()
     msdk_printf(MSDK_STRING("  -la           Use the look ahead bitrate control algorithm (LA BRC) for H.264 encoder. Supported only with -hw option on 4th Generation Intel Core processors. \n"));
     msdk_printf(MSDK_STRING("  -lad depth    Depth parameter for the LA BRC, the number of frames to be analyzed before encoding. In range [1,100]. \n"));
     msdk_printf(MSDK_STRING("                May be 1 in the case when -mss option is specified \n"));
-    msdk_printf(MSDK_STRING("  -la_ext       Use external LA plugin (compatible with h264 & hevc encoders)\n"));
+    msdk_printf(MSDK_STRING("  -la_ext       Use external LA plugin (compatible with h264 & hevc encoders). To use LA for HEVC encode, see readme_multi-transcode.pdf on the usage \n"));
```

Comment:
> readme_multi-transcode.pdf

There is no such file in the repository. There is [readme-multi-transcode_linux.md](https://github.com/Intel-Media-SDK/MediaSDK/blob/master/doc/samples/readme-multi-transcode_linux.md).

I also don't think that's a good idea to add references to readme documents to sample command line readme in the middle of the text for a single option. I think it is better to add a single note in the very beginning or in the very end of the list of options that there is a readme document available at the specified http addressed.

Reply:
@dvrogozh , updated the requested change. Can you please review & merge the request.

Reply:
Hi Naveen Kumar! 

Thank you for the patch!
Echo Dmitry R. ... I also think it's better to have a **full link** to readme-multi-transcode_linux.md in a single place (e.g. at the beginning/end of help output) and in places where we need to mention it, just mention short name.

Reply:
Hi Dmitry, 
I added full link to readme at the end of help output. Please review the latest commit c879b4c. Thanks